src/main/scala/edu/berkeley/ce/rockslicing/RockSlicer.scala

```diff
@@ -48,9 +47,19 @@ object RockSlicer {
       Block(center, faces.map(_.applyTolerance))
     }
 
-    // Convert the list of rock blocks to JSON and save this to a file
-    val jsonBlocks = squeakyClean.map(json.blockToMinimalJson)
-    jsonBlocks.saveAsTextFile("blocks.json")
+    // Convert list of rock blocks to requested output
+    if (inputs.toInequalities) {
+      // Convert the list of rock blocks to JSON and save this to a file
+      val jsonBlocks = squeakyClean.map(json.blockToMinimalJson)
+      jsonBlocks.saveAsTextFile("blocks.json")
+    }
+    if (inputs.toVTK) {
+      // Convert the list of rock blocks to JSON with vertices, normals and connectivity in format easily converted
+      // to vtk my rockProcessor module
+      val vtkBlocks = squeakyClean.map(BlockVTK(_))
+      val jsonVtkBlocks = vtkBlocks map JsonToVtk.blockVtkToMinimalJson
```

Comment:
Let's be consistent with syntax when we call `map` in these two cases. Above, you use `.map(json.blockToMinimalJson)` and here you use `map JsonToVtk.blockVtkToMinimalJson`. Either style would work fine.